x/bridge/keeper/keeper.go

```diff
@@ -12,23 +12,26 @@ import (
 )
 
 type Keeper struct {
-	cdc        codec.Codec
-	storeKey   storetypes.StoreKey
-	bankKeeper types.BankKeeper
-	evmKeeper  types.EvmKeeper
+	cdc          codec.Codec
+	storeKey     storetypes.StoreKey
+	bankKeeper   types.BankKeeper
+	evmKeeper    types.EvmKeeper
+	blockedAddrs map[string]bool
 }
 
 func NewKeeper(
 	cdc codec.Codec,
 	storeKey storetypes.StoreKey,
 	bankKeeper types.BankKeeper,
 	evmKeeper types.EvmKeeper,
+	blockAddrs map[string]bool,
```

Comment:
Let's be consitent and name this variable `blockedAddrs`

Reply:
yep, that's a typo.

Reply:
addressed in 34c05ba4c2ca492883883a4a4489e646382cdea4